### PR TITLE
acm4773 remove known issues

### PR DESCRIPTION
**Pull request by cadawson**
Link to issue: https://issues.redhat.com/browse/ACM-4773

### CODE REVIEW

File: release_notes/known_issues_governance.adoc [hunk 1]
```diff
@@ -86,27 +86,10 @@ When modifying an existing configuration policy, `DeleteAll` or `DeleteIfCreated
 [#policy-status-repeated-updates]
 == Policy status shows repeated updates when enforced
 
-If a policy is set to `remediationAction: enforce` and is repeatedly updated, the {product-title-short} console shows repeated violations with successful updates. This might happen in the following two cases:
-
-- Another controller or process is also updating the object with different values.
+If a policy is set to `remediationAction: enforce` and is repeatedly updated, the {product-title-short} console shows repeated violations with successful updates. This might happen when another controller or process is also updating the object with different values.
 +
 To resolve the issue, disable the policy and compare the differences between `objectDefinition` in the policy and the object on the managed cluster. If the values are different, another controller or process might be updating them. Check the `metadata` of the object to help identify why the values are different.
```
Comment (swopebe) [inline release_notes/known_issues_governance.adoc:92]: I am assuming this was not intentionally removed entirely?

**Review by swopebe — APPROVED** [review]
one question/comment

File: release_notes/known_issues_governance.adoc (hunk 1 above)
Comment (cadawson) [reply release_notes/known_issues_governance.adoc:92]: Yes, it was two possible situations. One was eliminated, so had a rework a bit.

**Comment by cadawson** [comment]
Thanks for the review!

**Pull request merged**